observability/src/test/java/io/grpc/observability/LoggingTest.java

```diff
@@ -17,55 +17,117 @@
 package io.grpc.observability;
 
 import static com.google.common.truth.Truth.assertThat;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.verifyNoInteractions;
+import static org.mockito.Mockito.when;
 
+import com.google.common.collect.ImmutableMap;
 import io.grpc.ManagedChannel;
 import io.grpc.ManagedChannelBuilder;
 import io.grpc.Server;
 import io.grpc.ServerBuilder;
+import io.grpc.internal.TimeProvider;
+import io.grpc.observability.interceptors.ConfigFilterHelper;
+import io.grpc.observability.interceptors.ConfigFilterHelper.MethodFilterParams;
 import io.grpc.observability.interceptors.InternalLoggingChannelInterceptor;
 import io.grpc.observability.interceptors.InternalLoggingServerInterceptor;
+import io.grpc.observability.interceptors.LogHelper;
 import io.grpc.observability.logging.GcpLogSink;
 import io.grpc.observability.logging.Sink;
 import io.grpc.testing.GrpcCleanupRule;
 import io.grpc.testing.protobuf.SimpleServiceGrpc;
 import java.io.IOException;
+import java.util.Map;
 import org.junit.Ignore;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
+import org.mockito.Mockito;
 
 @RunWith(JUnit4.class)
 public class LoggingTest {
 
   @Rule
   public final GrpcCleanupRule cleanupRule = new GrpcCleanupRule();
 
-  private static final String PROJECT_ID = "project-id";
+  private static final String PROJECT_ID = "PROJECT";
+  private static final Map<String, String> locationTags = ImmutableMap.of(
+      "project_id", "PROJECT",
+      "location", "us-central1-c",
+      "cluster_name", "grpc-observability-cluster",
+      "namespace_name", "default" ,
+      "pod_name", "app1-6c7c58f897-n92c5");
+  private static final Map<String, String> customTags = ImmutableMap.of(
+      "KEY1", "Value1",
+      "KEY2", "VALUE2");
 
   /**
    * Cloud logging test using LoggingChannelProvider and LoggingServerProvider.
    */
   @Ignore
```

Comment:
Why is it ignored?

Reply:
This test uses external Cloud Logging APIs, hence ignoring it.
Added comment explaining the same.